tests/migrations/test_autodetector.py

```diff
@@ -762,13 +738,11 @@ def content_file_name(key, **kwargs):
             (_content_file_name, ('other-file',), {}),
             (value.func, value.args, value.keywords)
         )
-
```

Comment:
the blank line here seems to be separating logical parts of the test so I wouldn't remove it